FreeRTOS_TCP_WIN.c

```diff
@@ -1183,9 +1183,9 @@
                         FreeRTOS_debug_printf( ( "lTCPWindowRxCheck[%d,%d]: seqnr %u exp %u (dist %d) SACK to %u\n",
                                                  ( int ) pxWindow->usPeerPortNumber,
                                                  ( int ) pxWindow->usOurPortNumber,
```

Comment:
A cast from `short` or `char` to `int` would not be necessary as the compiler will up-cast them automatically when ellipsis are used.